src/main/java/edu/ie3/datamodel/io/connectors/SqlConnector.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.connectors;
+
+import java.sql.*;
+import java.util.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Implements a DataConnector for a native SQL connection to a relational database. It was
+ * implemented with a PostgreSQL database in mind, so there might be dialect issues if used with
+ * other databases.
+ */
+public class SqlConnector implements DataConnector {
+  private static final Logger log = LogManager.getLogger(SqlConnector.class);
+
+  private final String jdbcUrl;
+  private final String userName;
+  private final String password;
+
+  /**
+   * Initializes a SqlConnector with the given JDBC url, username and password
+   *
+   * @param jdbcUrl the JDBC url, should start with "jdbc:postgresql://" and contain the database
+   *     name
+   * @param userName Name of the role used for authentication
+   * @param password Password for the role
+   */
+  public SqlConnector(String jdbcUrl, String userName, String password) {
+    this.jdbcUrl = jdbcUrl;
+    this.userName = userName;
+    this.password = password;
+  }
+
+  /**
+   * Initializes a SqlConnector with default parameters: <br>
+   * jdbcUrl = jdbc:postgresql://localhost:5432/simona <br>
+   * username = postgres <br>
+   * password = root <br>
+   */
+  public SqlConnector() {
+    this("jdbc:postgresql://localhost:5432/simona", "postgres", "root");
+  }
```

Comment:
same as in `CouchbaseConnector` - would remove this and make the actual parameters explicit when calling the parametrized constructor

```suggestion
```